Admin/FieldDescription.php

```diff
@@ -16,7 +16,7 @@
 class FieldDescription extends BaseFieldDescription
 {
     /**
-     * {@inheritdoc}
+     * Constructor.
```

Comment:
Sadly this page doesn't really reflect what is done in the symfony repository i've seen some pull request that add phpdoc to undocummented methods getting refused because they don't add any information.

An example is https://github.com/symfony/symfony/pull/16081